.gitignore

```diff
@@ -282,3 +282,4 @@ samples/openapi3/client/petstore/go/privatekey.pem
 
 ## OCaml
 samples/client/petstore/ocaml/_build/
+samples
```

Comment:
Thanks for the PR. Why add this line?

Reply:
Ah - I think that's a hang over from what I was using as an output directory. I'll remove it. Thanks